indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -382,8 +383,18 @@ protected void reduce(
           context.progress();
           final InputRow inputRow = index.formatRow(parser.parse(value.toString()));
```

Comment:
Can you move the try block to start up here? parser.parse can also throw ParseException